test/e2e/validation_test.go

```diff
@@ -503,3 +503,65 @@ func (tc *testContext) validateUpgradeableCondition(expected meta.ConditionStatu
 	}
 	return nil
 }
+
+// runWindowsJob creates a Windows Server Job with the given `name` and execution `command`. In addition, check for
+// errors based on `expectError` and is able to delete the job if specified by `deleteJob`
+func runWindowsJob(name, command string, expectError, deleteJob bool,
+	t *testing.T, testCtx *testContext) *batch.Job {
+	winJob, err := testCtx.createWindowsServerJob(name,
+		[]string{
+			testCtx.getPowerShellExe(), "-Command", command,
+		},
+		nil)
+	require.NoError(t, err, "could not create Windows tester job")
+	if deleteJob {
+		defer testCtx.deleteJob(winJob.Name)
+	}
```

Comment:
Adding this functionality doesnt seem too helpful, and is bloating the responsibilities of this function.
The function already returns the job that can be removed with `defer testCtx.deleteJob(job)`, but there are cases where the job needs to be cleaned up outside of the calling function.

Reply:
+1

Reply:
Agree, fixed in [e62e6c](https://github.com/openshift/windows-machine-config-operator/pull/777/commits/e62e6c5e00f900b70947e4ccb624aa7566a68228#diff-19a6ecdd2ef106d645523b5e6ddc22fd6ddfd1608477cd0b0fbd85113c8c245dR566)